ground/src/main/java/com/google/android/ground/persistence/remote/firebase/schema/SurveyConverter.kt

```diff
@@ -33,12 +33,21 @@ internal object SurveyConverter {
 
     val surveyFromProto = SurveyProto::class.parseFrom(doc, 1)
     val jobMap = jobs.associateBy { it.id }
+    val datasSharingTerms =
+      if (
```

Comment:
I think a more robust way of doing this would be to check `surveyFromProto.dataSharingTerms.type == DataSharingTerms.Type.DATA_SHARING_TYPE_UNSPECIFIED`, since it captures both the "unset message field" case and the "unset enum field" case.